setup.py

```diff
@@ -38,6 +48,9 @@
         'Topic :: Internet :: WWW/HTTP',
         'Topic :: Text Processing :: Markup :: HTML',
     ],
-    install_requires=['numpy', 'w3lib', 'six', 'cython'],
-    ext_modules=cythonize(extensions),
+    install_requires=['numpy', 'w3lib', 'six'],
+    extras_require={
+        'speedup': ['cython']
```

Comment:
If cython isn't installed it will build the c extension from the included `_htmlpage.pyx` file. If you're downloading from pypi and the `_htmlpage.c` file is included that will be used to create the extension instead.